### ISSUE TITLE
custom P2 theme

### ISSUE THREAD

**Issue by tjheffner** [phase2/frontend-docs#5]
mkdocs is a great service but the base theme is an eye-sore. please make a new theme that looks nice and is in a p2 color scheme. this theme will realistically get used on other p2 documentation, so make it look nice.

**Comment by grayside** [comment]
Yes please! http://docs.outrigger.sh is one example. We've pushed the limits of the navigation IA, so something to support more depth on the site would be nice.

**Comment by grayside** [comment]
Oops.

Mkdocs themes are the main form of customization, so there's some aggressive stuff you can do.

A few links to get things moving:
* http://www.mkdocs.org/user-guide/styling-your-docs/
* https://github.com/mkdocs/mkdocs/wiki/MkDocs-Themes
* https://squidfunk.github.io/mkdocs-material/

Hey a bootstrap theme. Say, frontending the frontend docs with apps/mkdocs?
